serenata_toolbox/speeches.py

```diff
@@ -0,0 +1,116 @@
+import os
+import pandas as pd
+import numpy as np
+import xml.etree.ElementTree as ET
+import urllib
+
+from datetime import datetime
+
+"""
+Usage:
+
+    range_start = '01/11/2016'
+    range_end = '30/11/2016'
+    output = 'data/2016-11-speeches.xz'
+
+    speeches = Speeches()
+    df = speeches.fetch(range_start, range_end)
+    speeches.write_file(output, df)
```

Comment:
To be easily used in other script ([example](https://github.com/datasciencebr/rosie/blob/master/rosie/main.py#L18-L25)) you could wrap this in a function, something like:

```python
def fetch_speeches(output, start, end):
    """
    :param output: (str) directory in which the output file will be saved
    :param start: (str) date in the formar dd/mm/yyyy
    :param end: (str) date in the formar dd/mm/yyyy
    """
    speeches = Speeches()
    df = speeches.fetch(start, end)
    speeches.write_file(output, df)
```

This way people can used it like this:

```python
from serenata_toolbox.speeches import fetch_speeches
fetch_speeches('01/01/1970', '31/12/2016')  # done ; )
```